help/destinations/catalog/crm/salesforce.md

```diff
@@ -158,66 +184,76 @@ Read [Activate profiles and segments to streaming segment export destinations](/
-    | --- | --- | --- |
-    |`person.name.lastName`|`LastName`| `Required`. Last name of the contact up to 80 characters. |    
-    |`person.name.firstName`|`FirstName`| The contact's first name up to 40 characters. |
-    |`personalEmail.address`|`Email`| The contact's email address. |
+    * Depending on the fields you want to export or update, add mappings between your XDM profile schema and [!DNL (API) Salesforce CRM]:
+        |Source Field|Target Field| Notes |
+        | --- | --- | --- |
+        |`IdentityMap: crmID`|`Identity: SalesforceId`|`Mandatory`|
+        |`xdm: person.name.lastName`|`Attribute: LastName`| `Mandatory`. Last name of the contact up to 80 characters. |    
+        |`xdm: person.name.firstName`|`Attribute: FirstName`| The contact's first name up to 40 characters. |
+        |`xdm: personalEmail.address`|`Attribute: Email`| The contact's email address. |
 
     * An example using these mappings is shown below:
     ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/crm/salesforce/mappings-contacts.png)
 
+    **Working with Leads**
+
     * If you are working with *Leads* within your segment, refer to the Object Reference in Salesforce for [Lead](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_lead.htm) to define mappings for the fields to be updated.
     * You can identify mandatory fields by searching for the word *Required*, which is mentioned in field descriptions in the link above.
-    * Depending on the fields you want to export or update, add mappings between your XDM profile schema and your [!DNL Salesforce CRM] instance:
-
-    | XDM Profile Schema | [!DNL Salesforce CRM] Instance | Notes |
-    | --- | --- | --- |
-    |`person.name.lastName`|`LastName`| `Required`. Last name of the contact up to 80 characters. |    
-    |`b2b.companyName`|`Company`| `Required`. The lead's company. |
-    |`personalEmail.address`|`Email`| The contact's email address. |
+    * Depending on the fields you want to export or update, add mappings between your XDM profile schema and [!DNL (API) Salesforce CRM]:
+        |Source Field|Target Field| Notes |
+        | --- | --- | --- |
+        |`IdentityMap: crmID`|`Identity: SalesforceId`|`Mandatory`|
+        |`xdm: person.name.lastName`|`Attribute: LastName`| `Mandatory`. Last name of the lead up to 80 characters. |    
+        |`xdm: b2b.companyName`|`Attribute: Company`| `Mandatory`. The lead's company. |
+        |`xdm: personalEmail.address`|`Attribute: Email`| The lead's email address. |
 
     * An example using these mappings is shown below:
     ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/crm/salesforce/mappings-leads.png)    
 
+When you have finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
 ### Schedule segment export and example {#schedule-segment-export-example}
```

Comment:
For the Schedule segment export and example section, please take into account all the comments that I left in the similar section in PR https://github.com/AdobeDocs/experience-platform.en/pull/174 . For brevity, I won't repeat the comments here.